src/main/java/com/ericsson/ei/notifications/EmailSender.java

```diff
@@ -83,13 +83,29 @@ public void sendEmail(MimeMessage message) throws NotificationFailureException {
      * */
     public MimeMessage prepareEmailMessage(String recipients, String mapNotificationMessage,
             String emailSubject) {
+        setDefaultValues();
         Set<String> emails = new HashSet<>();
         emails = extractEmails(recipients);
         String[] to = emails.toArray(new String[0]);
         MimeMessage message = prepareEmail(mapNotificationMessage, emailSubject, to);
         return message;
     }
 
+    /*
+     * This method assigns the default values of sender and subject of email notification if values
+     * at application.proporties are empty.
+     * */
+    private void setDefaultValues() {
+        if (sender.isEmpty() && subject.isEmpty()) {
```

Comment:
I think this can be simplified with only two if statements right? To first check sender and then check subject. A few less lines of code 😄